integration-tests/hazelcast-client/src/test/resources/classpathseed

```diff
@@ -0,0 +1,4 @@
+The purpose of this file is to have a non-empty test-classes directory that will
+be added to the test classpath and used by the Quarkus test extension for storing
+generated classes and other resources that should appear on the classpath during
+the test execution.
```

Comment:
I don't think this file is needed anymore. @ppalaga can you confirm?

Reply:
FYI I added it after having a look at `debezium-outbox` and `qpid-jms`, which still have these

Reply:
Yes, they do not seem to be needed anymore, @aloubyansky may want to confirm. Just try to remove them and check whether the tests are both executed and passing.

Reply:
I think it still is. It can easily be verified.

Reply:
Just to double-check... what would be the definition of "it works"?

We remove the file, run the build, and see if tests from the `*-integration-test` artifact run? or should I look for some generated test sources somewhere?

Reply:
Not worth the hassle, let's keep it if nobody is sure.